kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/actions/MultiFileConvertAction.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+package org.kitodo.mediaserver.core.actions;
+
+import java.io.File;
+import java.io.InputStream;
+import java.util.AbstractMap;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Pattern;
+import org.kitodo.mediaserver.core.api.IAction;
+import org.kitodo.mediaserver.core.api.IConverter;
+import org.kitodo.mediaserver.core.api.IExtractor;
+import org.kitodo.mediaserver.core.api.IMetsReader;
+import org.kitodo.mediaserver.core.api.IReadResultParser;
+import org.kitodo.mediaserver.core.config.FileserverProperties;
+import org.kitodo.mediaserver.core.config.MetsProperties;
+import org.kitodo.mediaserver.core.conversion.FileEntry;
+import org.kitodo.mediaserver.core.db.entities.Work;
+import org.kitodo.mediaserver.core.exceptions.ConversionException;
+import org.kitodo.mediaserver.core.util.MediaServerUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.StringUtils;
+
+/**
+ * An action to convert all master images to one PDF file.
+ */
+public class MultiFileConvertAction implements IAction {
```

Comment:
This is a bit dubious. On the one hand, the class name suggest, that the class could be used for any multi file conversion. On the other hand, the code suggests it is only for PDF. Wouldn't it be better to call the class something with PDF?